test/conformance/chainsaw/exceptions/exclude-image-reference/policy.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: check-image
+spec:
+  validationFailureAction: Enforce
+  background: false
+  webhookTimeoutSeconds: 30
+  failurePolicy: Fail
```

Comment:
Those fields are deprecated. Can we use the new ones?
`webhookConfiguration.timeoutSeconds` and `webhookConfiguration.failurePolicy`.